lib/mindwendel/lanes.ex

```diff
@@ -70,12 +70,9 @@ defmodule Mindwendel.Lanes do
 
   """
   def get_lanes_for_brainstorming_with_labels_filtered(id) do
-    brainstorming = Brainstormings.get_brainstorming!(id)
+    brainstorming = Brainstormings.get_bare_brainstorming!(id)
 
-    filter_label =
-      if length(brainstorming.filter_labels_ids) > 0,
-        do: %{filter_labels_ids: brainstorming.filter_labels_ids},
-        else: %{}
+    filter_label = %{filter_labels_ids: brainstorming.filter_labels_ids}
```

Comment:
I found that the `if` here doesn't matter and we could just deal with the empty list in the functions instead. Small drive by :)

Reply:
nice :D